src/index.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import HelloWorld from './components/HelloWorld';
+import NotFound from './components/NotFound';
+import Success from './components/Success'
 
-ReactDOM.render(<HelloWorld />, document.getElementById('app'));
\ No newline at end of file
+ReactDOM.render(<Success />, document.getElementById('app'));
```

Comment:
the correct component for render is App, and this "js" just need this, Success and NotFound will be use in each page

Reply:
Got it! i will work on this pages!